components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3182,6 +3182,21 @@ def definition_body(self):
 """ % args)
 
 
+class CGInitTypeHoldedMethod(CGAbstractMethod):
+    def __init__(self, descriptor):
+        assert descriptor.interface.hasInterfaceObject()
+        CGAbstractMethod.__init__(self, descriptor, 'InitTypeHolded',
+                                  'void', [], unsafe=True, templateArgs=['TH: TypeHolderTrait'])
+
+    def definition_body(self):
+        properties = PropertyArrays(self.descriptor)
+        define = ""
```

Comment:
Since PropertyArrays contains a bunch of objects derived from PropertyDefiner, we can get the name of the relevant Rust array from the `variableNames` property. Then for each element in the array, we can replace the pointer member of the object with a name that is based on the corresponding entry in the `regular` list for that PropertyDefiner. Does that clear up the issue?